lib/gen_stage/streamer.ex

```diff
@@ -13,25 +13,32 @@ defmodule GenStage.Streamer do
         x, {acc, counter} -> {:cont, {[x | acc], counter - 1}}
       end)
 
-    {:producer, continuation, Keyword.take(opts, [:dispatcher, :demand])}
+    {:current_stacktrace, [_info_call | stack]} = Process.info(self(), :current_stacktrace)
```

Comment:
I think you have to this in `start_link`, otherwise it will mostly contain information about the gen_server and inner stacktrace.

Reply:
Ah yes it did lol